databricks/koalas/frame.py

```diff
@@ -947,6 +948,63 @@ def groupby(self, by):
         gp = self._sdf.groupby(by)
         return PandasLikeGroupBy(self, gp, None)
 
+    def isin(self, values):
+        """
+        Whether each element in the DataFrame is contained in values.
+
+        :param values: iterable or dict
```

Comment:
can you just use the numpy docs format, similar to Pandas?

https://github.com/pandas-dev/pandas/blob/v0.24.2/pandas/core/frame.py#L7857-L7944

Reply:
Okay! 
btw, many methods use restructuredtext style  like `reset_index()`.